web-frontend/src/pages/TransferCoins/SendCoins/ContactList.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  MassaToken,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  Selector,
+  Identicon,
+} from '@massalabs/react-ui-kit';
+import { formatStandard } from '../../../utils/MassaFormating';
+import { useResource } from '../../../custom/api';
+import { AccountObject } from '../../../models/AccountModel';
+
+import { toMAS } from '@massalabs/massa-web3';
+
+function AccountSelect({ ...props }) {
+  const { modalAccounts, setModalAccounts, setRecipient, account } = props;
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+  const otherAccounts = accounts.filter(
+    (acnt: AccountObject) => acnt.nickname !== account.nickname,
+  );
+
+  function getFormattedBalance(account: AccountObject): string {
```

Comment:
No need one function (getFormattedBalance) that only wraps another one. I would place these two casts (toMas, toNumber) under the **formatStandard** or a variation of it and use them direct in place of **getFormattedBalance**